src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,121 @@
 package core.basesyntax;
 
-public class MyHashMap<K, V> implements MyMap<K, V> {
+public class MyHashMap<K,V> implements MyMap<K,V> {
+    private static final int DEFAULT_ARRAY_LENGTH = 16;
+    private static final double LOAD_FACTOR = 0.75;
+    private int elementsAmount;
+    private Node<K,V> [] valuesArray;
```

Comment:
don't include data type in the variable's name, Java is a strongly typed language
```suggestion
    private Node<K,V> [] values;
```